src/commonMain/kotlin/io/github/koalaplot/core/line/LinePlot.kt

```diff
@@ -514,6 +512,30 @@ private fun <X, Y> XYGraphScope<X, Y>.generateArea(
     }
 }
 
+private fun DrawScope.fillRectangle(
```

Comment:
Does this draw the rectangle differently than DrawScope.drawRect? Why couldn't that be used?